coverage_comment/coverage.py

```diff
@@ -67,13 +67,15 @@ def compute_coverage(num_covered: int, num_total: int) -> decimal.Decimal:
     return decimal.Decimal(num_covered) / decimal.Decimal(num_total)
 
 
-def get_coverage_info(merge: bool, coverage_path: pathlib.Path) -> Coverage:
+def get_coverage_info(
+    merge: bool, coverage_path: pathlib.Path
+) -> tuple[dict, Coverage]:
     try:
         if merge:
             subprocess.run("coverage", "combine", path=coverage_path)
 
-        json_coverage = subprocess.run(
-            "coverage", "json", "-o", "-", path=coverage_path
+        json_coverage = json.loads(
```

Comment:
(I tend to try and limit things that happen in a try statement to the statements that may raise. But it's not very important, as in this case, it's clear the json.loads can't interact with the subprocess error.)